src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -6,10 +6,10 @@
 import java.util.Properties;
 
 public class ConnectionUtil {
-    private static final String URL = "YOUR DATABASE URL";
-    private static final String USERNAME = "YOUR USERNAME";
-    private static final String PASSWORD = "YOUR PASSWORD";
-    private static final String JDBC_DRIVER = "YOUR DRIVER";
+    private static final String URL = "jdbc:mysql://localhost:3306/taxi_service";
+    private static final String USERNAME = "root";
+    private static final String PASSWORD = "11111111";
+    private static final String JDBC_DRIVER = "com.mysql.cj.jdbc.Driver";
```

Comment:
You forget add SQL table script for tables car and cars_drivers into file init_db.sql in resources